dataprofiler/profilers/profile_builder.py

```diff
@@ -1424,82 +1507,95 @@ def tqdm(l):
                 cols=len(data.columns))
 
         # Format the data
-        notification_str = "Finding the Null values in the columns..."        
-        if pool and len(new_cols) > 0:
+        notification_str = "Finding the Null values in the columns..."
+        if pool and new_cols:
```

Comment:
I'm not sure why new_cols exists or why the notification addendum is dependent on new cols. Can we get rid of this?

Reply:
Thats a good point, I never understood the need for this condition, will remove